packages/tradershub/src/constants/password.ts

```diff
@@ -4,5 +4,13 @@ export const passwordRegex = {
     hasSymbol: /\W/,
     hasUpperCase: /[A-Z]/,
     isLengthValid: /^.{8,25}$/,
-    isPasswordValid: /^(?=.*[a-z])(?=.*\d)(?=.*[A-Z])[!-~]{8,25}$/,
+    isPasswordValid: /^(?=.*[a-z])(?=.*\d)(?=.*[A-Z])(?=.*[_@?!/#])[!-~]{8,25}$/,
```

Comment:
<img width="1000" alt="Screenshot 2024-03-20 at 9 50 20 AM" src="https://github.com/binary-com/deriv-app/assets/75345074/a8d2ff06-9494-471c-92d3-596193545d46">


Based on the backend JSON schema, the old regex is correct. 

```
            "pattern": "^(?=.*[a-z])(?=.*[0-9])(?=.*[A-Z])[ -~]{8,25}$",
```

Please get someone to update the schema. We shouldn't have different validation in frontend

Reply:
the old regex is correct indeed, `[ -~]` is checking for special symbol already